lib/fixer/sqs_client.rb

```diff
@@ -0,0 +1,44 @@
+# -*- encoding: utf-8 -*-
+
+require 'aws-sdk-core'
+
+module Fixer
+  class SqsClient
+
+    include Configuration
+
+    attr_accessor *Fixer::Configuration.keys
+
+    def create_job(job, opts = {})
+      job[:job][:id] ||= SecureRandom.uuid
+      job[:job][:client_id] ||= client_id
+
+      sqs_options = options.merge(opts)
+      sqs_queue = sqs_options[:queue]
+      conn = sqs(sqs_options[:aws])
+      queue_url = conn.get_queue_url(queue_name: sqs_queue).queue_url
+      conn.send_message(queue_url: queue_url, message_body: job.to_json)
+      job
+    end
+
+    def sqs(options = nil)
+      @sqs ||= Aws::SQS::Client.new(options)
+    end
+
+    def sqs=(sqs)
+      @sqs = sqs
+    end
+
+    def initialize(options = {})
+      apply_options(options)
+      yield(self) if block_given?
+    end
+
+    def apply_options(options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `apply_options(*)` if you want the method to accept any arguments but don't care about them.